css/civicrm.css

```
@@ -2684,9 +2684,15 @@ div.crm-container form {
   width: 9em;
 }
 
-.crm-container input.dateplugin::placeholder,
-.crm-container input.crm-form-date::placeholder,
-.crm-container input.crm-form-time::placeholder {
+.crm-container input.crm-placeholder-icon::placeholder {
```

I think this should be `crm-placeholder-right` here and below.

```suggestion
.crm-container input.crm-placeholder-right::placeholder {
```

Also, are there three separate blocks because IE needs it to be that way?

---

According to what I've read, all the major browsers support `::placeholder` except MS is still lagging behind, and also inconsistent about which one of the two brand-prefixes it wants.